### PR TITLE
add 168. Excel Sheet Column Title

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 20m
Please check my code.

### CODE REVIEW

**Review by ryuji0123 — CHANGES_REQUESTED** [review]
Empty file.

**Comment by kobakobashu** [comment]
I fixed empty file.  Please check it.

File: 168_excel-sheet-column-title.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    def convertToTitle(self, columnNumber: int) -> str:
+        """
+        time: O(x)  x : 26^x < columnNumber < 26^(x + 1)
+        space: O(x)
+        """
+        alp = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+        column_title_list = []
+        cur_quotient = columnNumber
+        cur_remainder = 0
+        while 26 < cur_quotient:
```
Comment (ryuji0123) [inline 168_excel-sheet-column-title.py:18]: avoid magic number: 26

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I avoided magic number: 26.  Please check my code.

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request closed without merging**